app/models/work.rb

```diff
@@ -84,6 +84,11 @@ class Work < Kithe::Work
   attr_json :file_creator, :string
   attr_json :admin_note, :text, array: true, default: -> { [] }
 
+
+  OCR_REQUESTED_OPTIONS = %w{yes no}.freeze
+  attr_json :ocr_requested, :string, default: "no"
+  validates :ocr_requested, inclusion: { in: OCR_REQUESTED_OPTIONS, allow_blank: false }
```

Comment:
Should we make this `:boolean` instead of `:string`?  Then it's already either true or false, and no need to validate further. 

I am not sure if there are any unanticipated problems with using :boolean in json, which we don't do very much? But let's try it?

Reply:
Thanks! If we need to convert to String later, we can one way or another -- I _think_ that the :string value might even automatically be able to read the boolean properly as `"true"` , or `"false"`, and if you switched to string and then just went through every object and read it in then `save`d it again, it would convert to String.

Reply:
Great - it's a boolean now!